src/test/java/core/basesyntax/StorageTest.java

```diff
@@ -0,0 +1,88 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class StorageTest {
+    private Storage storage;
+
+    @BeforeEach
+    public void setUp() {
+        storage = new Storage();
+    }
+
+    @Test
+    public void addFruit_increasesQuantityCorrectly() {
+        storage.addFruit("apple", 10);
+        storage.addFruit("apple", 5);
+
+        assertEquals(15, storage.getQuantity("apple"));

```

Comment:
if you want to test the method addFruit of the Storage don't use getQuantity of Storage for comparing results. Use methods of collection directly. The same for other methods of this class